### ISSUE TITLE
Rethink Sidebar in terms of plane english

### ISSUE THREAD

**Issue by bobbilee19** [TBD54566975/developer.tbd.website#411]
Thinking from the perspective of a developer who knows nothing about web5 SSI DIDs etc.....

I'm seeing terms off the bat (within Learn and Build) that are foreign to me (DWN and DIDs), so now I'm just clicking around into pages trying to figure out the what, why, and ultimately how.

I'm proposing we use plane english like "Identity" for DIDs & "Storage" for DWN...or any other words someone else can think of that are simple and clear.

Would love collab on this and a brainstorm session or comments from others on this issue.

**Comment by bobbilee19** [comment]
@ALRubinger I don't recognize that project, has the project named changed?

**Comment by angiejones** [comment]
torn here because they do need to become familiar with the terms so are we doing them any favors by not calling the thing the thing?

perhaps use icons in front of the sidebar titles to give indication of what it's related to. Like an server icon to represent DWNs

**Comment by bobbilee19** [comment]
Ah that's a good idea angie! Thoughts on this @dayhaysoos @kirahsapong ?

**Comment by EbonyLouis** [comment]
yes, that goes to some of the feedback we got yesterday. We want them to know the terms but we want it to resonate with something they already know as well. So I like your idea Nick of having both, instead of just an icon.

**Comment by angiejones** [comment]
i like that idea Nick

**Comment by Smartmind12** [comment]
@angiejones Is this issue open to work on?
